third_party/libjpeg-turbo/turbojpeg.c

```diff
@@ -44,17 +44,13 @@
 #include "./cdjpeg.h"
 
 #ifdef STARBOARD
-#include "starboard/client_porting/poem/strings_poem.h"
-#include "starboard/client_porting/poem/string_poem.h"
+#include "starboard/common/log.h"
 #include "starboard/configuration.h"
-#include "starboard/client_porting/poem/stdio_poem.h"
 
 #include "starboard/once.h"
 #include "starboard/thread.h"
-#else
-#include <stdio.h>
 #endif
-
+#include <stdio.h>
```

Comment:
Is this needed? It's not in the imported tree:
https://chromium.googlesource.com/chromium/deps/libjpeg_turbo/+/ed683925/turbojpeg.c#44

[same for `ctype.h` on the next line, incidentally]

Reply:
For me the build succeeds without including `stdio.h` and `ctype.h`.